wsdlgo/encoder_test.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 func LoadDefinition(t *testing.T, filename string, want error) *wsdl.Definitions {
-	f, err := os.Open(filepath.Join("testdata", filename))
+	f, err := os.Open(GetDefinitionFilePath(filename))
```

Comment:
what's wrong with just `filepath.Join` here? the other function only calls that anyway